src/main/java/io/stargate/sgv2/jsonapi/config/DatabaseLimitsConfig.java

```diff
@@ -30,7 +30,7 @@ public interface DatabaseLimitsConfig {
    * Default setting for {@link #maxCollections()}}: maximum number of Collections allowed to be
    * created per Database, regardless of whether enough Indexes are available.
    */
-  int DEFAULT_MAX_COLLECTIONS = 5;
+  int DEFAULT_MAX_COLLECTIONS = 10;
```

Comment:
Sorry for not being clear, my intent was not that we update the defaults here but that we override configured values for specific environments.

Reply:
Thanks, reverted.